### ISSUE TITLE
Unable to buy / sell

### ISSUE THREAD

**Issue by codeguru1983** [0xProject/0x-launch-kit#43]
Hi guys,
Similar issue in your erc20 demo, now on my deployment.

I've managed to install it but there's no way I can do any buy / sell orders on market or limit.

![image](https://user-images.githubusercontent.com/58865041/85539005-14a73d00-b61e-11ea-8d58-fb188b8ae320.png)
although I have ETH, WETH, ZRX in my wallet, I've unlocked them, there's no way. I click on buy button but nothing happens:

also in the console I can see loads of errors:
![image](https://user-images.githubusercontent.com/58865041/85539423-7c5d8800-b61e-11ea-879c-30cc0d84f4fe.png)

Do I have to enable in the firewall these ports or what should I do ?

Also, is there any proper tutorial on how to add my own tokens?

I use ropsten test network

Thank you!

**Comment by codeguru1983** [comment]
Also if you can please advise on how we can add a chart onto this exchange will be great and maybe a list with the pairs also will make it look like a proper exchange thank you!

**Comment by elterremoto182** [comment]
I have the same issue.

**Comment by callybendt** [comment]
Yes me to I can do any buy / sell orders on market or limit some help

**Comment by codeenlab** [comment]
I am also getting same error. Please provide any solution
